BSP/Inc/BSP_ADC.h

```diff
@@ -0,0 +1,13 @@
+#ifndef __BSP_ADC_H
+#define __BSP_ADC_H
+
+//function that returns the percentage pressed of the accelerator pedal
+float ADC_Percent_Accel_Read();
+
+//function that returns the percentage pressed of the accelerator pedal
+float ADC_Percent_Brake_Read();
+
+// Initialize the ADC module
+void ADC_Init(void);
```

Comment:
The functions we want to expose through this API are as follows:

`BSP_ADC_Init()`
`BSP_ADC_Accel_GetMillivoltage()`
`BSP_ADC_Brake_GetMillivoltage()`

The two bottom functions should return the value in millivolts for each respective ADC.

Reply:
With whatever new functions you want to add and expose through the API, make sure to use the ```BSP_ADC_``` prefix to the functions.